protos/flyteidl/admin/node_execution.proto

```diff
@@ -171,10 +185,10 @@ message NodeExecutionGetDataRequest {
 // Response structure for NodeExecutionGetDataRequest which contains inputs and outputs for a node execution.
 message NodeExecutionGetDataResponse {
     // Signed url to fetch a core.LiteralMap of node execution inputs.
-    UrlBlob inputs                             = 1;
+    UrlBlob inputs                             = 1  [deprecated = true];
 
     // Signed url to fetch a core.LiteralMap of node execution outputs.
-    UrlBlob outputs                            = 2;
+    UrlBlob outputs                            = 2  [deprecated = true];
 
     // Optional, full_inputs will only be populated if they are under a configured size threshold.
```

Comment:
is this really optional now that urlblob is deprecated?

Reply:
updated